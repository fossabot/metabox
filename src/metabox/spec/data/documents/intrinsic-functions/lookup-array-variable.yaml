Metabox:
  Description: Join arrays
  
  Resources:
    PropertyArrayJoin:
      Type: "metabox::resource"
      Properties:
        ArrayJoin: 
          Fn::Join:
            - ":"
            - - a
              - b
              - c 
        ArrayShortJoin: { "Fn::Join": [ ",", [1,2,3] ] }