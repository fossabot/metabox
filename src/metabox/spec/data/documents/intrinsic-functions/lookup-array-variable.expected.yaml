Metabox:
  Description: Join arrays
  
  Resources:
    PropertyArrayJoin:
      Type: "metabox::resource"
      Properties:
        ArrayJoin: 'a:b:c'
        ArrayShortJoin: '1,2,3'